File: JS/quiz.js
document.write("Welcome!");
var questions = [
    // this question array is just question object
    // that store question and answer 
    {
        prompt: "what color are apples\n(a)Red\n(b)Green\n(c)Purple",
        answer: "a"
    },

    {
        prompt: "What color are Bananas\n(a)Teal\n\(b)Yellow\n(c)Orange",
        answer: "b"

    },

    {
        prompt: "What color are strawberries\n(a)Red\n\(b)Green\n(c)Blue",
        answer: "a"

    }
]

var score = 0;
for (var i=0; i<questions.length; i++){
    var response = window.prompt(questions[i].prompt);
    if(response == questions[i].answer){
        score++;
        alert("you right!!!");
    }
    else{
        alert("you got it wrong!");
    }
}

alert("you got "+ score + "/"+ questions.length);

File: Python/quiz.py
from multiprocessing.connection import answer_challenge
from question_class import Question


questions_show = [
    "what color are apples?\n(a) Red/Green\n(b) Purple\n(c) Orange\n\n",

    "what color are bananas?\n(a) Teal\n(b) Magenta\n(c) Yellow\n\n",
    
    "what color are strawberries?\n(a) Yellow\n(b) Red\n(c) Blue\n\n"
]


# creat question object
questions = [
    Question(questions_show[0],"a"),
    #this is like Question(prompt, answer)
    Question(questions_show[1],"c"),
    Question(questions_show[2],"b"),
]


def run_test(questions):
    score = 0
    for question in questions:
        user_response = input(question.prompt)
        if user_response == question.answer:
            score +=1
    print("you got "+ str(score)+ "/"+ str(len(questions_show)))
        
            
run_test(questions)